### ISSUE TITLE
Late: 35; Underflow: 1; Overflow: 0; message when I am connecting galaxy M33 5G with enb through USRP x310 

### ISSUE THREAD

**Issue by sruthi81** [srsran/srsRAN_Project#971]
Here I am sharing the problem when I am connecting my ue with gnb. I am running Open5GS and srsRAN in same machine and usrp x310 is used as RU. 
![Screenshot from 2024-12-05 11-35-54](https://github.com/user-attachments/assets/3c93b494-dc0a-4c07-a7b0-96eae3527481)
 "Late: 35; Underflow: 1; Overflow: 0;" message.

Please help me to solve this issue.

**Comment by pgawlowicz** [comment]
could you share your configs and log?

**Comment by sruthi81** [comment]
Yes Sir.



# This configuration file example shows how to configure the srsRAN Project gNB to allow srsUE to connect to it. 
# This specific example uses ZMQ in place of a USRP for the RF-frontend, and creates an FDD cell with 10 MHz bandwidth. 
# To run the srsRAN Project gNB with this config, use the following command: 
#   sudo ./gnb -c gnb_zmq.yaml

amf:
  addr: 127.0.0.5                 # The address or hostname of the AMF.
  bind_addr: 127.0.0.5             # A local IP that the gNB binds to for traffic from the AMF.

ru_sdr:
  device_driver: uhd                # The RF driver name.
  device_args: addr=192.168.10.2 #tx_port=tcp://127.0.0.1:2000,rx_port=tcp://127.0.0.1:2001,base_srate=23.04e6 # Optionally pass arguments to the selected RF driver.
  #sync:  gpsdo                   # Specify the sync source used by the RF.
#device_args: tx_port=tcp://192.168.0.105:2000,rx_port=tcp://192.168.0.114:2001,base_srate=23.04e6 # Optionally pass arguments to the selected RF driver.
  clock: internal  
  srate: 23.04                      # RF sample rate might need to be adjusted according to selected bandwidth.
  tx_gain: 30                       # Transmit gain of the RF might need to adjusted to the given situation.
  rx_gain: 30                       # Receive gain of the RF might need to adjusted to the given situation.

cell_cfg:
  dl_arfcn: 368500                  # ARFCN of the downlink carrier (center frequency).
  band: 3                           # The NR band.
  channel_bandwidth_MHz: 20         # Bandwith in MHz. Number of PRBs will be automatically derived.
  common_scs: 15                    # Subcarrier spacing in kHz used for data.
  plmn: "00101"                     # PLMN broadcasted by the gNB.
  tac: 7                            # Tracking area code (needs to match the core configuration).
  pdcch:
    common:
      ss0_index: 0                  # Set search space zero index to match srsUE capabilities
      coreset0_index: 12            # Set search CORESET Zero index to match srsUE capabilities
    dedicated:
      ss2_type: common              # Search Space type, has to be set to common
      dci_format_0_1_and_1_1: false # Set correct DCI format (fallback)
  prach:
    prach_config_index: 1           # Sets PRACH config to match what is expected by srsUE

cu_cp:
  inactivity_timer: 7200            # Sets the UE/PDU Session/DRB inactivity timer to 7200 seconds. Supported: [1 - 7200].

log:
  filename: /tmp/gnb.log            # Path of the log file.
  all_level: info                   # Logging level applied to all layers.
  hex_max_size: 0

pcap:
  mac_enable: false                 # Set to true to enable MAC-layer PCAPs.
  mac_filename: /tmp/gnb_mac.pcap   # Path where the MAC PCAP is stored.
  ngap_enable: false                # Set to true to enable NGAP PCAPs.
  ngap_filename: /tmp/gnb_ngap.pcap # Path where the NGAP PCAP is stored.


I cant open the amf log file, it shows permission denied.

**Comment by pgawlowicz** [comment]
you can find logs at /tmp/gnb.log

**Comment by sruthi81** [comment]
ok

**Comment by sruthi81** [comment]
Sir, 
Please help me to resolve this issue
 Thanks

**Comment by pgawlowicz** [comment]
please try to run this script:
https://github.com/srsran/srsRAN_Project/blob/main/scripts/srsran_performance

and try again.

**Comment by sruthi81** [comment]
![Uploading Screenshot from 2025-01-13 10-12-44.png…]()
Sir, is it correct?

**Comment by sruthi81** [comment]
![Screenshot from 2025-01-13 10-12-44](https://github.com/user-attachments/assets/67e21430-fa02-44c2-adbc-a96185edd018)

**Comment by sruthi81** [comment]
same result I got after running the script.

**Comment by sruthi81** [comment]
please help me to resolve the issue.

**Comment by sruthi81** [comment]
![Screenshot from 2025-01-15 10-08-30](https://github.com/user-attachments/assets/a0af4da8-535e-46c6-9f93-039b328fd45f)
The gnb started but frame size issue is there how to resolve.

**Comment by pgawlowicz** [comment]
could you share your config? Which OS version do you use?

**Comment by sruthi81** [comment]
amf:
  addr: 127.0.0.5                 # The address or hostname of the AMF.
  bind_addr: 127.0.0.5             # A local IP that the gNB binds to for traffic from the AMF.

ru_sdr:
  device_driver: uhd                # The RF driver name.
  device_args: addr=192.168.10.2 #tx_port=tcp://127.0.0.1:2000,rx_port=tcp://127.0.0.1:2001,base_srate=23.04e6 # Optionally pass arguments to the selected RF driver.
  #sync:  gpsdo                   # Specify the sync source used by the RF.
#device_args: tx_port=tcp://192.168.0.105:2000,rx_port=tcp://192.168.0.114:2001,base_srate=23.04e6 # Optionally pass arguments to the selected RF driver.
  clock: internal  
  srate: 23.04                      # RF sample rate might need to be adjusted according to selected bandwidth.
  tx_gain: 30                       # Transmit gain of the RF might need to adjusted to the given situation.
  rx_gain: 30                       # Receive gain of the RF might need to adjusted to the given situation.

cell_cfg:
  dl_arfcn: 368500                  # ARFCN of the downlink carrier (center frequency).
  band: 3                           # The NR band.
  channel_bandwidth_MHz: 20         # Bandwith in MHz. Number of PRBs will be automatically derived.
  common_scs: 15                    # Subcarrier spacing in kHz used for data.
  plmn: "00101"                     # PLMN broadcasted by the gNB.
  tac: 7                            # Tracking area code (needs to match the core configuration).
  pdcch:
    common:
      ss0_index: 0                  # Set search space zero index to match srsUE capabilities
      coreset0_index: 12            # Set search CORESET Zero index to match srsUE capabilities
    dedicated:
      ss2_type: common              # Search Space type, has to be set to common
      dci_format_0_1_and_1_1: false # Set correct DCI format (fallback)
  prach:
    prach_config_index: 1           # Sets PRACH config to match what is expected by srsUE

cu_cp:
  inactivity_timer: 7200            # Sets the UE/PDU Session/DRB inactivity timer to 7200 seconds. Supported: [1 - 7200].

log:
  filename: /tmp/gnb.log            # Path of the log file.
  all_level: info                   # Logging level applied to all layers.
  hex_max_size: 0

pcap:
  mac_enable: false                 # Set to true to enable MAC-layer PCAPs.
  mac_filename: /tmp/gnb_mac.pcap   # Path where the MAC PCAP is stored.
  ngap_enable: false                # Set to true to enable NGAP PCAPs.
  ngap_filename: /tmp/gnb_ngap.pcap # Path where the NGAP PCAP is stored.

**Comment by sruthi81** [comment]
and I am using Ubuntu 22.04.5 LTS

**Comment by sruthi81** [comment]
please help me to resolve this issue

**Comment by sruthi81** [comment]
please help me to resolve the issue

**Comment by newt2105** [comment]
hi @sruthi81, did you solve this problem?

**Comment by sruthi81** [comment]
Yes the internet issue solved but getting the underflow overflow issue and not able to trace the gnb. please help me to resolve the issue.

**Comment by newt2105** [comment]
hi @sruthi81, I have same problem but haven't fixed yet
